### ISSUE TITLE
input type="image" shows no label in Accessibility Properties

### ISSUE THREAD

**Issue by jnurthen** [GoogleChrome/accessibility-developer-tools#279]
See testcase at http://output.jsbin.com/xikuvurero

The Accessibility Properties Tab shows an empty label for this image button.
